interface/main/tabs/edit_menu.php

```diff
@@ -82,8 +82,7 @@
     <link href="js/jsoneditor/jsoneditor.css" rel="stylesheet" type="text/css">
     <script src="js/jsoneditor/jsoneditor.js"></script>
 
```

Comment:
Why are these lines marked with red? Is it because of the comment `/* Include our required headers */` in this file above ? Please check.